tests/test_qaoa_objective_sk.py

```diff
@@ -0,0 +1,187 @@
+###############################################################################
+# // SPDX-License-Identifier: Apache-2.0
+# // Copyright : JP Morgan Chase & Co
+###############################################################################
+import numpy as np
+import pytest
+from functools import partial
+from qiskit_aer import AerSimulator
+from itertools import combinations
+from qokit.sk import sk_obj, get_sk_terms, get_random_J
+from qokit.maxcut import maxcut_obj
+from qokit.qaoa_objective_sk import get_qaoa_sk_objective
+from qokit.utils import brute_force, precompute_energies
+from qokit.parameter_utils import get_sk_gamma_beta
+from qokit.fur import get_available_simulators, get_available_simulator_names
+
+
+rng = np.random.default_rng(seed=42)
+
+qiskit_backend = AerSimulator(method="statevector")
+SIMULATORS = get_available_simulators("x") + get_available_simulators("xyring") + get_available_simulators("xycomplete")
+simulators_to_run_names = get_available_simulator_names("x") + ["qiskit"]
+simulators_to_run_names_no_qiskit = get_available_simulator_names("x")
+
+
+def test_sk_obj(N=5):
```

Comment:
Please make how N is set consistent. In some tests it is an argument, and in some tests it is set on the first line of the test

Reply:
Done!